container/common/fsHandler.go

```
@@ -119,16 +149,42 @@ func (fh *realFsHandler) trackUsage() {
 	}
 }
 
+func (fh *realFsHandler) trackInodeUsage() {
+	fh.updateInodeUsage()
```

don't ignore the error